vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -650,6 +679,12 @@ The absence of `RestrictedProperties` denotes all properties are accessible usin
         <Property Name="Updatable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
           <Annotation Term="Core.Description" String="Entities can be updated" />
         </Property>
+        <Property Name="Upsertable" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Entities can be upserted" />
+        </Property>
+        <Property Name="CollectionUpdate" Type="Capabilities.CollectionUpdateType" Nullable="false">

```

Comment:
>Nullable="false" [](start = 83, length = 16)

Should not be nullable=false

Reply:
Should be nullable

Reply:
Removed and moved properties of CollectionUpdateType here